libraries/azurerm_storage_account.rb

```diff
@@ -0,0 +1,41 @@
+# frozen_string_literal: true
+
+require 'azurerm_resource'
+
+class AzurermStorageAccout < AzurermSingularResource
+  name 'azurerm_storage_account'
+  desc 'Verifies settings for a Azure Storage Account'
+  example <<-EXAMPLE
+    describe azurerm_storage_account(resource_group: resource_name, name: 'default') do
+      it { should exist }
+      its('secure_transfer_enabled') { should be true }
```

Comment:
I'd consider making a custom matcher for things that are booleans.

Something like:
```
it { should have_secure_transfer_enabled }
```